tests/integration/integration_test.py

```diff
@@ -116,7 +116,7 @@ def test_main_success(monkeypatch: MonkeyPatchFixture, init_events, self_monitor
     assert Counter(self_monitoring.dynatrace_connectivities) == {DynatraceConnectivity.Ok: 3}
     assert self_monitoring.processing_time > 0
     assert self_monitoring.sending_time > 0
-    assert self_monitoring.sent_log_entries == 12
+    assert self_monitoring.sent_log_entries == 16
```

Comment:
I have no knowledge in this field - could you please explain why did you make this change?

Reply:
There are 5 events combined from 5 logs so there are 25 logs. 5 are omitted due to too old timestamp, 4 are omitted because of too long content.